deploy/example_catalog/cr-app-cdh632cm.json

```diff
@@ -0,0 +1,415 @@
+                        "hbase-master",
+                        "hbase-thrift",
+                        "hbase-rest"
+                    ],
+                    "roleID": "hbase"
+                },
+                {
+                    "serviceIDs": [
+                        "cdh-scm-agent",
+                        "ssh",
+                        "hive-metastore",
+                        "hive-server",
+                        "webhcat"
+                    ],
+                    "roleID": "hive"
+                },
+                {
+                    "serviceIDs": [
+                        "cdh-scm-agent",
+                        "ssh",
+                        "impala"
+                    ],
+                    "roleID": "impala"
+                },
+                {
+                    "serviceIDs": [
+                        "cdh-scm-agent",
+                        "ssh",
+                        "hue",
+                        "oozie",
+                        "spark-history"
+                    ],
+                    "roleID": "apps"
+                },
+                {
+                    "serviceIDs": [
+                        "cdh-scm-agent",
+                        "ssh",
+                        "yarn-nm",
+                        "hbase-worker"
+                    ],
+                    "roleID": "worker"
+                },
+                {
+                    "serviceIDs": [
+                        "cdh-scm-agent",
+                        "ssh"
+                    ],
+                    "roleID": "gateway"
+                }
+            ],
+            "selectedRoles": [
+                "cmserver",
+                "primary",
+                "hive",
+                "worker",
+                "gateway",
+                "hbase",
+                "impala",
+                "apps"
+            ]
+        },
+        "defaultImageRepoTag": "docker.io/bluedata/cdh632multi:1.0",
+        "label": {
+            "name": "CDH 6.3.2 multirole 7x",
+            "description": "CDH 6.3.2 with YARN and HBase support. Includes Hive, Hue, Impala and Spark."
+        },
+        "distroID": "bluedata/cdh632multi",
+        "version": "1.0",
+        "configSchemaVersion": 7,
+        "services": [
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 7180,
+                    "path": "/"
+                },
+                "id": "cdh-scm-srvr",
+                "label": {
+                    "name": "Cloudera Manager"
+                }
+            },
+            {
+                "id": "cdh-scm-srvr-db",
+                "label": {
+                    "name": "Cloudera manager database"
+                }
+            },
+            {
+                "id": "cdh-scm-agent",
+                "label": {
+                    "name": "Cloudera manager agent"
+                }
+            },
```

Comment:
no endpoints for these?

Reply:
@swamibluedata We are declaring all the services with or without end points in the "services" key. The id of all the services is being also referred in "serviceIDs" key within "roleServices" key where we specify which service is running on which role.